examples/stable/cassandra/cassandra-blueprint.yaml

```diff
@@ -0,0 +1,183 @@
+apiVersion: cr.kanister.io/v1alpha1
+kind: Blueprint
+metadata:
+  name: k10-cassandra-bp-xk5c2
```

Comment:
@viveksinghggits Can we change the blueprint name to something generic and meaningful like (cassandra-blueprint)?

Reply:
yes I agree, I will do that.